sacrerouge/metrics/meteor.py

```diff
@@ -133,17 +137,20 @@ def add_subparser(self, parser: argparse._SubParsersAction):
 
     @overrides
     def run(self, args):
-        commands = [
-            f'mkdir -p {DATA_ROOT}/metrics/METEOR',
-            f'cd {DATA_ROOT}/metrics/METEOR',
-            f'wget https://www.cs.cmu.edu/~alavie/METEOR/download/meteor-1.5.tar.gz',
-            f'tar xzvf meteor-1.5.tar.gz'
-        ]
-        command = ' && '.join(commands)
-
-        process = Popen(command, shell=True)
-        process.communicate()
-        if process.returncode == 0:
-            print('METEOR setup success')
-        else:
-            print('METEOR setup failure')
+        output_dir = Path(f'{DATA_ROOT}/metrics/METEOR')
+        output_dir.mkdir(exist_ok=True, parents=True)
+
+        output_file = output_dir / 'meteor-1.5.tar.gz'
+
+        if not output_file.exists():
+            r = requests.get('https://www.cs.cmu.edu/~alavie/METEOR/download/meteor-1.5.tar.gz', stream=True)
```

Comment:
There is functionality to download a file from a url here
https://github.com/danieldeutsch/sacrerouge/blob/534e2fde8943c987bffbfbe15d85350646ae9422/sacrerouge/common/util.py#L20